src/app/link/link.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+	selector: 'app-link',
+	templateUrl: './link.component.html',
+	styleUrls: ['./link.component.scss']
+})
+export class LinkComponent {
+	@Input() color: string;
+	@Input() text: string;
+	@Input() url: string;
+
+	setData(data: { color?: string, text: string; url: string; }): void {
+		data = Object.assign({ color: 'primary', text: '', url: '' }, data);
+
+		this.color = data.color;
```

Comment:
Color should be hooked up or removed until it is actually used by the generated docs